File: class9/48/Person1.java
package com.demo.scala06;

public class Person1 implements Comparable<Person1>{
    //定义属性
    private String name;
    private int age;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public Person1(String name, int age) {
        this.name = name;
        this.age = age;
    }

    @Override
    public int compareTo(Person1 o) {
        //升序
        return this.age - o.age;
    }
}

File: class9/46/Person1.scala
package test.scala01

//定义类
class Person1 {
  //定义姓名 年龄
  var name: String = _
  var age: Int = _
}

//继承App特质 可以不写main
object Test extends App{
  val p = new Person1
  p.name = "tom"
  p.age = 18
  println(p.name)
}
